src/providers/ldap/sdap_id_op.c

```diff
@@ -608,6 +608,10 @@ static void sdap_id_op_connect_done(struct tevent_req *subreq)
 
             default:
                 /* do not attempt to retry on errors like ENOMEM */
+                DEBUG(SSSDBG_TRACE_FUNC,
+                      "Marking the backend (%p) offline [%d]: %s\n",
```

Comment:
I hate to be that guy who complains about a comment, but instead of a be pointer, I think dereferencing be to `domain->name` might be more descriptive..